### PR TITLE
NOISSUE - Add experimental conditional build

### PR DESCRIPTION
**Pull request by darkodraskovic**

### CODE REVIEW

File: src/app/pages/pages-menu.ts [hunk 1]
```diff
@@ -32,25 +23,40 @@ export const MENU_ITEMS: NbMenuItem[] = [
...
-        title: 'Gateways',
-        icon: 'hard-drive-outline',
-        link: '/pages/services/gateways',
-      },
-    ],
-  },
 ];
+
+if (environment.experimental) {
+  MENU_ITEMS.push(
```
Comment (manuio) [inline src/app/pages/pages-menu.ts:29]: I would simply replace all Menu. With push() this is going to change the structure

**Review by manuio — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -14,7 +14,9 @@
     "conventional-changelog": "conventional-changelog",
     "start": "ng serve --proxy-config=proxy-config.json",
     "build": "ng build",
-    "build:prod": "npm run build -- --prod --aot",
+    "build:prod": "npm run build -- --prod",
```
Comment (manuio) [inline package.json:17]: Keep `--aot`

File: package.json [hunk 3]
```diff
@@ -14,7 +14,9 @@
...
+    "experimental": "ng serve --proxy-config=proxy-config.json --configuration=experimental",
+    "experimental:prod": "npm run build --configuration=experimental-prod",
```
Comment (manuio) [inline package.json:19]: Add `--aot` here

**Review by manuio — CHANGES_REQUESTED** [review]

File: src/app/pages/pages-menu.ts [hunk 4]
```diff
@@ -32,25 +78,6 @@ export const MENU_ITEMS: NbMenuItem[] = [
...
-      {
-        title: 'Gateways',
-        icon: 'hard-drive-outline',
-        link: '/pages/services/gateways',
-      },
-    ],
-  },
 ];
+
+
```
Comment (manuio) [inline src/app/pages/pages-menu.ts:83]: Remove this blank line

**Review by manuio — CHANGES_REQUESTED** [review]

**Review by manuio — APPROVED** [review]
LGTM

**Review by dusanb94 — APPROVED** [review]

**Pull request merged**